Model/Quote/Payment/GetPaymentProfileId.php

```diff
@@ -10,9 +10,15 @@
 use SubscribePro\Service\Address\AddressInterface;
 use SubscribePro\Service\PaymentProfile\PaymentProfileInterface;
 use Swarming\SubscribePro\Gateway\Config\ConfigProvider as SubscribeProConfigProvider;
+use Swarming\SubscribePro\Gateway\Config\ApplePayConfigProvider;
 
 class GetPaymentProfileId
 {
+    const PAYMENT_METHODS_THAT_STORE_TOKENS_LOCALLY = [
```

Comment:
Please, set const `private` and name `SPRO_PAYMENT_METHODS`

Reply:
done